pennylane_lightning/lightning_qubit/_state_vector.py

```diff
@@ -246,14 +250,36 @@ def _apply_lightning_controlled(self, operation):
                 False,
             )
 
-    def _apply_lightning(self, operations):
+    def _apply_lightning_midmeasure(self, operation: MidMeasureMP, mid_measurements: dict):
```

Comment:
could it be `None`?

```suggestion
    def _apply_lightning_midmeasure(self, operation: MidMeasureMP, mid_measurements: dict = None):
```

Reply:
I'd rather not because the sole purpose is to apply `measure` and this should return an MCM.

Reply:
hmm.. right, right! then let's update the docs.